relentless/ensemble.py

```diff
@@ -8,14 +8,59 @@
 from . import core
 
 class RDF(core.Interpolator):
+    """Constructs a :py:class:`Interpolator` object interpolating through an RDF
+    function :math:`g(r)`, and creates an `nx2` array with columns as `r` and `g`.
+
+    Parameters
+    ----------
+    r : array_like
+        1-d array of r values, that must be continually increasing.
+    g : array_like
+        1-d array of g values.
+
+    """
     def __init__(self, r, g):
         super().__init__(r,g)
         self.table = np.column_stack((r,g))
 
 class Ensemble(object):
-    def __init__(self, types, T, P=None, V=None, mu={}, N={}, kB=1.0, conjugates=None):
-        self.types = tuple(types)
-        self.rdf = core.PairMatrix(self.types)
+    """Constructs a thermodynamic ensemble for a set of types with specified parameters.
+
+    Parameters
+    ----------
+    T : float or int
+        Temperature of the system.
+    P : float or int
+        Pressure of the system (defaults to `None`).
+    V : float or int
+        Volume of the system (defaults to `None`).
+    mu : `dict`
+        The chemical potential for each specified type (defaults to empty `dict`).
+    N : `dict`
+        The number of particles for each specified type (defaults to empty `dict`.
+    kB : float
+        Boltzmann constant (defaults to 1.0).
+
+    Raises
+    ------
+    ValueError
+        If either `P` or `V` are not set.
+    ValueError
+        If either `mu` or `N` are not set for each type.
+    TypeError
+        If all values of N are not integers.
+    ValueError
+        If the conjugates are not set, and both `P` and `V` are set.
+    ValueError
+        If the conjugates are not set, and both `mu` and `N` are set for each type.
+
+    """
+    def __init__(self, T, P=None, V=None, mu={}, N={}, kB=1.0, conjugates=None):
```

Comment:
Remove the conjugates parameter.
```suggestion
    def __init__(self, T, P=None, V=None, mu={}, N={}, kB=1.0):
```